### DISCUSSION TITLE
Add shortcut ctrl + k for search column

### DISCUSSION THREAD

**Discussion by 3xploiton3** [pradyunsg/furo#750] [Ideas] [answered]
example:
![Screenshot Capture - 2024-01-28 - 11-51-36](https://github.com/pradyunsg/furo/assets/19517680/0d313548-5875-4477-a0e9-3c03ac749db7)

**Comment by pradyunsg** [comment, marked as answer]
This shortcut overrides the browser default for navigating to the browser search, which is why I didn't add this.
